pallav/fashionTech/pallavBanka/parkingAssignment/Services/ParkingCharge.java

```diff
@@ -0,0 +1,23 @@
+package fashionTech.pallavBanka.parkingAssignment;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ParkingCharge {
+    private Long ParkingCharges(VehicleConstant.VehicleType vehicleType, Date inTime, Date outTime) {
+        Map<VehicleConstant.VehicleType, Integer> charges = new HashMap();
+        charges.put(VehicleConstant.VehicleType.twowheeler, 20);
+        charges.put(VehicleConstant.VehicleType.fourwheeler, 40);
+        charges.put(VehicleConstant.VehicleType.sixwheeler, 80);
```

Comment:
Move the Map to ChargesDao, and expose method there to get perHourCharges By vehicle Type